packages/core/src/shipping/strategies/stripe-upe/stripe-upe-shipping-strategy.ts

```diff
@@ -0,0 +1,83 @@
+import { PaymentMethodActionCreator } from '../../../payment';
+
+import { AddressRequestBody } from '../../../address';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
+import { StripeElements, StripeElementType, StripeEventType, StripeScriptLoader, StripeUPEClient } from '../../../payment/strategies/stripe-upe';
+import ConsignmentActionCreator from '../../consignment-action-creator';
+import { ShippingInitializeOptions, ShippingRequestOptions } from '../../shipping-request-options';
+import ShippingStrategy from '../shipping-strategy';
+
+export default class StripeUpeShippingStrategy implements ShippingStrategy {
+    private _stripeUPEClient?: StripeUPEClient;
+    private _stripeElements?: StripeElements;
+    private sendData: any;
+
+    constructor(
+        private _store: CheckoutStore,
+        private _stripeUPEScriptLoader: StripeScriptLoader,
+        private _consignmentActionCreator: ConsignmentActionCreator,
+        private _paymentMethodActionCreator: PaymentMethodActionCreator
+    ) {}
+
+    updateAddress(address: AddressRequestBody, options?: ShippingRequestOptions): Promise<InternalCheckoutSelectors> {
+        return this._store.dispatch(
+            this._consignmentActionCreator.updateAddress(address, options)
+        );
+    }
+
+    selectOption(optionId: string, options?: ShippingRequestOptions): Promise<InternalCheckoutSelectors> {
+        return this._store.dispatch(
+            this._consignmentActionCreator.selectShippingOption(optionId, options)
+        );
+    }
+
+    async initialize(options: ShippingInitializeOptions): Promise<InternalCheckoutSelectors> {
+        const { methodId = "", stripeupe: { container, onChangeShipping, availableCountries } = {} } = options;
+        const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(methodId, { params: { method: 'card' } }));
+        const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow('card', methodId);
+        const { initializationData: { stripePublishableKey, stripeConnectedAccount } } = paymentMethod;
+
+        if (!paymentMethod || !paymentMethod.initializationData.stripePublishableKey || !paymentMethod.clientToken) {
+                throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+            }
+
+        this._stripeUPEClient = await this._stripeUPEScriptLoader.getStripeClient(stripePublishableKey, stripeConnectedAccount);
+
+        const appearance = {};
+
+        this._stripeElements = this._stripeUPEScriptLoader.getElements(this._stripeUPEClient, {
+                clientSecret: paymentMethod.clientToken,
+                appearance: appearance,
```

Comment:
if you would like to keep the constant you can use the object literal property value shorthand like this
```suggestion
                appearance,
```

or you can pass directly the empty object since you don't modify it and it has no values
```suggestion
                appearance: {},
```